app/assets/javascripts/components/NewsletterSticky.js

```diff
@@ -53,6 +53,10 @@ define([
     this.closeButton = this.$el.find(this.config.closeClassSelector);
     $window = $(window);
 
+    var focusable = this.$el.find('a[href], area[href], input:not([disabled], [type=hidden]), select:not([disabled], [hidden]), textarea:not([disabled], [hidden]), button:not([disabled], [hidden]), iframe, object, embed, *[tabindex], *[contenteditable]');
```

Comment:
We follow the airbnb style guide for Js, which has jshint config for limiting it to 100 characters. So yep, bit long - I'll change this.